<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QrestMainWindowClass</class>
 <widget class="QMainWindow" name="QrestMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Qrest</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/pixmaps.qrc">
    <normaloff>:/icons/pix/qresticon.png</normaloff>:/icons/pix/qresticon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="tempoInputGroupBox">
          <property name="title">
           <string>Tempo Input</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="tempoInputHorizontalLayout">
             <item>
              <widget class="QLabel" name="tempoLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Temp&amp;o  : </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>tempoEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tempoEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>28</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>60</width>
                 <height>28</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Enter tempo in BPM and hit 'enter' or use mouse wheel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="tapButton">
               <property name="toolTip">
                <string>Click in sync with beats until steadiness indicator goes green</string>
               </property>
               <property name="text">
                <string>T&amp;AP</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Variants</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="plainRadio">
             <property name="text">
              <string>&amp;Plain</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="dottedRadio">
             <property name="text">
              <string>&amp;Dotted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="tripletRadio">
             <property name="text">
              <string>&amp;Triplet</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="midiGroupbox">
        <property name="title">
         <string>MIDI</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QRadioButton" name="freewheelRadio">
           <property name="text">
            <string>Of&amp;f</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="syncRadio">
           <property name="text">
            <string>MIDI Clock &amp;Sync</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="triggerRadio">
           <property name="text">
            <string>TapTempo trig&amp;ger</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Delays and LFOs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>ms</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Hz</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="wholePeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/whole.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="wholeLfoEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="halfPeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/half.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="halfLfoEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="quarterPeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/quarter.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="quarterLfoEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>ms</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Hz</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="eighthPeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/eighth.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="eighthLfoEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="sixteenthPeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/sixteenth.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="sixteenthLfoEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="thirtySecondPeriodEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../resources/pixmaps.qrc">:/notes/pix/thirtysecond.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="thirtySecondLfoEdit">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>28</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>60</width>
               <height>28</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../resources/pixmaps.qrc">
     <normaloff>:/icons/pix/quit.png</normaloff>:/icons/pix/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit Qrest</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../../resources/pixmaps.qrc">
     <normaloff>:/icons/pix/information.png</normaloff>:/icons/pix/information.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qrest...</string>
   </property>
   <property name="statusTip">
    <string>About Qrest</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../../resources/pixmaps.qrc">
     <normaloff>:/icons/pix/help.png</normaloff>:/icons/pix/help.png</iconset>
   </property>
   <property name="text">
    <string>Online &amp;Help</string>
   </property>
   <property name="statusTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../../resources/pixmaps.qrc">
     <normaloff>:/icons/pix/preferences.png</normaloff>:/icons/pix/preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="statusTip">
    <string>Open preferences dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tempoEdit</tabstop>
  <tabstop>tapButton</tabstop>
  <tabstop>plainRadio</tabstop>
  <tabstop>dottedRadio</tabstop>
  <tabstop>tripletRadio</tabstop>
  <tabstop>wholePeriodEdit</tabstop>
  <tabstop>halfPeriodEdit</tabstop>
  <tabstop>quarterPeriodEdit</tabstop>
  <tabstop>eighthPeriodEdit</tabstop>
  <tabstop>sixteenthPeriodEdit</tabstop>
  <tabstop>thirtySecondPeriodEdit</tabstop>
  <tabstop>wholeLfoEdit</tabstop>
  <tabstop>halfLfoEdit</tabstop>
  <tabstop>quarterLfoEdit</tabstop>
  <tabstop>eighthLfoEdit</tabstop>
  <tabstop>sixteenthLfoEdit</tabstop>
  <tabstop>thirtySecondLfoEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/pixmaps.qrc"/>
 </resources>
 <connections/>
</ui>
